04_data-viz.qmd

```diff
@@ -1,76 +1,81 @@
 
-ggplot2 is an R package for data visualization that was developed during Hadley Wickham's graduate studies at Iowa State University. ggplot2 is formalized in ["A Layered Grammar of Graphics"](https://vita.had.co.nz/papers/layered-grammar.html) by Hadley Wickham, which was published in the Journal of Statistical Software in 2010. 
+ggplot2 is an R package for data visualization that was developed during Hadley Wickham's graduate studies at Iowa State University. ggplot2 is formalized in *A Layered Grammar of Graphics* by Hadley Wickham @layered-grammar.
 
-The grammar of graphics, originally by Leland Wilkinson, is a theoretical framework that breaks all data visualizations into their component pieces. With the layered grammar of graphics, Wickham extends Wilkinson's grammar of graphics and implements it in R. The cohesion is impressive and the theory flows to the code which informs the data visualization process in a way not reflected in any other data viz tool.  
+The grammar of graphics, originally by Leland Wilkinson, is a theoretical framework that breaks all data visualizations into their component pieces. With the layered grammar of graphics, Wickham extends Wilkinson's grammar of graphics and implements it in R. The cohesion is impressive, and the theory flows to the code which informs the data visualization process in a way not reflected in any other data viz tool.  
 
 There are **eight** main ingredients to the grammar of graphics. We will work our way through the ingredients with many hands-on examples. 
 
-### `r kableExtra::text_spec("Exercise 0", color = "#1696d2")`
+::: callout
 
-**`r kableExtra::text_spec("Step 1:", color = "#55b748")`** Open your `.Rproj`.
+#### [`r paste("Exercise", exercise_number)`]{style="color:#1696d2;"}
 
-**`r kableExtra::text_spec("Step 2:", color = "#55b748")`** Create a new `.R` script in your directory called `03_data-visualization.R`. 
+```{r}
+#| echo: false
 
-### `r kableExtra::text_spec("Exercise 1", color = "#1696d2")`
+exercise_number <- exercise_number + 1
+```
 
-**`r kableExtra::text_spec("Step 1:", color = "#55b748")`** **Type** (don't copy & paste) the following code below `library(tidyverse)` in `03_data-visualization.R`.
+1. Open your `.Rproj`.
+2. Create a new `.R` script in your directory called `03_data-visualization.R`. 
+3.  **Type** (don't copy & paste) the following code below `library(tidyverse)` in `03_data-visualization.R`.
 
 ```
 ggplot(data = storms) + 
   geom_point(mapping = aes(x = pressure, y = wind))
 ```
+4. Add a comment above the ggplot2 code that describes the plot we created. 
+:::
 
-**`r kableExtra::text_spec("Step 2:", color = "#55b748")`** Add a comment above the ggplot2 code that describes the plot we created. 
+## Eight Ingredients in the Grammar of Graphics:
 
-\vspace{0.25in}
+### Data 
 
 :::{.callout-note}
-**1 Data** are the values represented in the visualization.
+**Data** are the values represented in the visualization.
 :::
```

Comment:
These should be callout-tip to align with the definition sections elsewhere in the notes.

Reply:
Ok I thought hierarchy didn't look as good, but I will change back!

Reply:
Changed!